core/templates/components/forms/schema-based-editors/schema-based-unicode-editor.component.html

```diff
@@ -15,7 +15,7 @@
 
   <span *ngIf="!getCodingMode() && !getRows()">
     <input type="text"
-           class="form-control display-inline"
+           class="form-control display-inline exploration-id-input"
```

Comment:
Custom classes should start with `oppia-...`

Reply:
done